demos/README.md

```diff
@@ -1,75 +1,219 @@
-  ```sh
-  $ npm run stylelint
-  $ npx stylelint [options] <dir|file|glob>
-  ```
+```sh
+$ npm run stylelint
+$ npx stylelint [options] <dir|file|glob>
+```
 
 * To make a check of the .html files you can execute either of the following two commands:
-  ```sh
-  $ npm run linthtml
-  $ html-validate [options] <dir|file|glob>
-  ```
+```sh
+$ npm run linthtml
+$ html-validate [options] <dir|file|glob>
+```
 
 In case you want to automatically apply the possible fixes, just add the --fix option in the desired command, the second command found in each specific block of commands, or in those explained in the README file at the root of the project.
 
+## File manifest
+
+### Important files and folder structure
+
+The next tree you will see represents the structure of this project in major terms. 
+
+```bash
+demos
+├── technoloy-1
+│   ├── editor-1
+|   |   ├── ...
+|   |   └── package.json
+│   ├── editor-2
+|   ├── ...
+|   └── editor-m
+├── technology-2
+│   ├── editor-1
+│   ├── editor-2
+|   ├── ...
+|   └── editor-m
+├── ...
+├── technology-n
+└── README.md
+```
+
+There is a set of `n` supported `technologies` and its `m` supported `editors`, which will be detailed below. 
+
+Every editor has a package.json for the editor dependencies and, relying on the folder you are in, you will see one kind of structure or another following the technlology requirements and implementation.
+
+### Supported editors
+
+This is the list of editors with a supported custom wiris plugins especially created for the editors:
+
+- `ckeditor4`,
+- `ckeditor5`,
+- `froala2`,
+- `froala3`,
+- `generic`,
+- `tinymce4`,
+- `tinymce5`;
+
+### Technologies with a demo
+
+- `html5`
+- [Comming soon] `React`
+- [Comming soon] `Angular`
+
+## Running tests
+
+Automatic test will be coming soon.
+
+## Known bugs
+
+- `CKeditor5` has problems working with lerna, because the ck5 architecture prohibits having duplicate dependencies, but there is a temporal solution.
+
+- `Froala3` launches a reference error that doesn’t affect the proper performance of the demo.
+
+We are working to improve the user experience and fix the errors previously detailed. :smile:
+
+## Change log
+
+- An `improvement` in the project `structure` has been implemented.
+
+- Improvements have been made to the functioning of .css, .html, and .js files, creating a `resource folder` containing all the common code between the demos.
+
+- `Duplicate` configuration `files` have been `removed`.
+
+- Technical demos project `documentation` has been `improved`.
+
+## News
+
+- :tada: Now there is a demo for froala version 3 with `react` technology. 
+
+## Documentation
+
+- [npm](https://docs.npmjs.com/)
+
+- [Lerna](https://github.com/lerna/lerna#readme) 
+
+- [Mathtype by Wiris](http://www.wiris.com/mathtype)
+
 ## License
 
+Copyright © 2010-2020 [WIRIS](http://www.wiris.com). Released under the [MIT License](../LICENSE).
 
 ## Contact information
 team.support.europe@wiris.com 
+
```

Comment:
This would have more to do with the code of conduct we want to include in the root README. Perhaps it would be good to make a reference to that part of the document when created to avoid duplication

Reply:
Okay, you can then leave that for when we update the root README if you want.